build-new.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
```

Comment:
I don't understand why to create a new .sh file instead of updating the existing one? It would be nice for people to still be able to invoke a build script via command line and not have it diverge over time from the one CI was running. I see the pod install is happening elsewhere now, maybe we can use another ENV var or argument to disable it and then keep just one file?

Reply:
This is a leftover from the development of the diff. If we want to do this I will replace the old build.sh before landing.

Reply:
No time like the present! Are you anticipating a lag between merging the PR and actually running CI on CircleCI?